kedro/framework/cli/utils.py

```diff
@@ -468,6 +472,11 @@ def _split_params(ctx, param, value):
     return conf
 
 
+def _split_load_versions(ctx, param, value):
+    lv_tuple = _get_values_as_tuple([value])
```

Comment:
Do we need the `[value]` here rather than `value`? Elsewhere we call `_get_values_as_tuple` just with a single string. 

Also do we need the `if/else` here? Wouldn't `_reformat_load_versions` work the same just passing the empty `value` into it directly?

After we've removed the old `load_version` we should definitely just move `_reformat_load_versions` code into this instead of having two separate functions.

Reply:
Cool, that's what I suspected about `multiple=True` 👍  

> The `if/else` isn't needed as `_reformat_load_versions` would still work - the idea was to prevent calling another method if unnecessary. Which approach would be considered more pythonic?

Overall I think without the `if/else`. The performance overhead of calling another function here is very small compared to the (also small) additional complexity of branching in the code. It's a small thing, but I'd only skip calling the function here if it was particularly expensive.